src/app/login/page.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import { FC, useState } from 'react';
+import Link from 'next/link';
+
+const LoginForm: FC = () => {
+  const [userData, setUserData] = useState({
+    email: '',
+    password: '',
+  });
+
+  return (
+    <section className="sec-cont login-container">
+      <div>
+        <h1>Inloggning</h1>
+        <form action="">
+          <label htmlFor="email">Email</label>
+          <input
+            onChange={(e) => {
+              e.preventDefault();
+              setUserData((prevData) => {
+                return {
+                  ...prevData,
+                  email: e.target.value,
+                };
+              });
+            }}
+            className="form-input-field"
+            type="text"
+            id="email"
+          />
+          <label htmlFor="password">Password</label>
+          <input
+            onChange={(e) => {
+              e.preventDefault();
```

Comment:
The prevent default doesn't go here, it has to be in the form using the onSubmit event.

Reply:
yes, of course!